gateway/controller_test.go

```diff
@@ -47,21 +46,70 @@ var _ = ginkgo.Describe("Controller", func() {
 		testRouter.HandleFunc("/schema/ns1/hash", func(w http.ResponseWriter, r *http.Request) {
 			w.Write([]byte("123"))
 		})
+		testRouter.HandleFunc("/assignment/ns1/hash/0", func(w http.ResponseWriter, r *http.Request) {
+			w.Write([]byte("123"))
+		})
+		testRouter.HandleFunc("/assignment/ns1/assignments/0", func(w http.ResponseWriter, r *http.Request) {
+			w.Write([]byte(`
+{  
```

Comment:
nit: job/topic/cluster names are too Uber specific.. let's fuzzy  it a bit..

Reply:
done